source/mainnet/smart-contracts/onboarding-guide-solana-developers/overview.rst

```diff
@@ -0,0 +1,112 @@
+.. _solana-overview:
+
+========
+Overview
+========
+
+This overview outlines the main differences of account models and smart contract functionality between Solana and Concordium.
+
+Accounts
+========
+
+Solana accounts are somewhat similar to files and can hold arbitrary data.
+Some accounts are marked as `executable` and contain program code.
+These accounts themselves are immutable, but programs stored in their data can modify data in accounts they own.
+
+Concordium, on the other hand, has a strict separation between user accounts and smart contracts.
+Both accounts and smart contracts have CCD balances associated with them.
+Moreover, smart contracts can have state -- structured data associated with the contract.
+Unlike Solana, this data can be updated directly only by the smart contract code it belongs to.
+The only way of interacting with the contract state from outside is through the interface that the smart contract provides.
+This interface consists of entrypoints.
+
+Another important difference is that transferring CCD to a non-existing account or interacting with a non-existing smart contract address on Concordium will fail.
+
+.. _solana-programs:
+
+Smart contracts and programs
+============================
+
+Concordium `smart contracts` serve the same purpose as `programs` on Solana.
```

Comment:
Is this the right format for a link?

Reply:
Yeah, I meant to just use emphasis here, I'll fix it.